x-pack/plugins/infra/public/components/asset_details/tabs/processes/process_row_charts.tsx

```diff
@@ -42,8 +47,27 @@ export const ProcessRowCharts = ({ command }: Props) => {
     <EuiEmptyPrompt iconType="warning" title={<EuiText>{failedToLoadChart}</EuiText>} />
   ) : isLoading ? (
     <EuiLoadingChart />
-  ) : (
+  ) : hasCpuData ? (
     <ProcessChart timeseries={response!.cpu} color={Color.color2} label={cpuMetricLabel} />
+  ) : (
+    <EuiFlexGroup
+      gutterSize="xs"
+      alignItems="center"
+      justifyContent="center"
+      responsive={false}
+      css={css`
+         {
+          height: 140px;
+        }
+      `}
+    >
+      <EuiFlexItem grow={false}>
+        <EmptyPlaceholder icon={IconChartLine} />
+      </EuiFlexItem>
+      <EuiFlexItem grow={false}>
+        <CpuNotAvailableExplanationTooltip />
+      </EuiFlexItem>
+    </EuiFlexGroup>
```

Comment:
Suggestion: we could try to improve the alignment of the tooltip icon, to be placed right next to the `no results found` text:

<img width="495" alt="image" src="https://github.com/elastic/kibana/assets/2767137/b7b5abc1-5350-4ba6-ae83-057552a757c7">

```suggestion
    <EmptyPlaceholder
      css={css`
         {
          height: 140px;
        }
      `}
      icon={IconChartLine}
      message={
        <EuiFlexGroup
          gutterSize="xs"
          alignItems="center"
          justifyContent="center"
          responsive={false}
        >
          <EuiFlexItem grow={false}>
            <FormattedMessage id="charts.noDataLabel" defaultMessage="No results found" />
          </EuiFlexItem>
          <EuiFlexItem grow={false}>
            <CpuNotAvailableExplanationTooltip />
          </EuiFlexItem>
        </EuiFlexGroup>
      }
    />
```

Reply:
Thanks for the suggestion! Done ✅